schematic/models/validate_attribute.py

```diff
@@ -1147,6 +1152,7 @@ def type_validation(
                     )
                     if vr_errors:
                         errors.append(vr_errors)
+                    # It seems impossible to get warnings with type rules
```

Comment:
Should this have any follow-up tech debt ticket?

Reply:
Yes I think so. I wanted to mention it here to see if I misunderstood something first.

Reply:
@andrewelamb I'm not sure where I'm seeing where it's impossible, could you explain in case I missed something? The default message for all the type rules is `error` but for every validation rule a user could specify in their data model to specifically raise `warnings` or `error` regardless of the default level

Reply:
@GiaJordan that's doesn't appear to be true. See [here.](https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2645262364/Data+Model+Validation+Rules#Type-Validation-Type)

"Use the type checker by setting the ‘Validation Rules’  to one of the types listed below. No additional specifiers are allowed. "

If you look at the code in the validator above that is confirmed. Unless the val_rule is one of ["num", "str", "int", "float"] you'll just get empty errors and warnigns back.

Reply:
@andrewelamb that does look like the current behavior, I don't think that's intended though, but that would probably be another discussion. Message level specifications are allowed in the rule definitions in `validation_rule_info()`

Reply:
I'll start a jira issue then.

Reply:
[issue](https://sagebionetworks.jira.com/browse/FDS-2506)